Source/UI/MLCardFormField/MLCardFormField.swift

```diff
@@ -278,7 +278,7 @@ internal extension MLCardFormField {
 
     func isValid() -> Bool {
         guard let value = getValue() else { return false }
```

Comment:
deberiamos llamar `.trimmingCharacters(in: .whitespaces)` dentro de `getValue()` para que todos los llamados ya obtengan el string sin los espacios en blanco.

Reply:
el campo ya hace trim cuando pierde foco.